### ISSUE TITLE
model = Sequential() model.add(Embedding(10000, embedding_size, input_length=max_words)) model.add(Conv1D(64, 3, padding='same')) model.add(Conv1D(32, 3, padding='same')) model.add(Flatten()) model.add(Dropout(0.2)) model.add(Dense(512,activation='relu')) model.add(Dropout(0.2)) model.add(Dense(1,activation='softmax’))

### ISSUE THREAD

**Issue by pvlakshmi** [keras-team/keras#12559]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by jvishnuvardhan** [comment]
@pvlakshmi Is there any issue here? If there is any issue, please provide more details about the issue, fill [template](https://github.com/keras-team/keras/issues/new/choose) and provide trace of the bug. Please let us know what is the backend you are using here. Thanks!

**Comment by jvishnuvardhan** [comment]
@pvlakshmi I noticed you had opened another issue. I am closing this as it is a duplicate issue. Thanks!
